jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/category/CategoryDetailView.java

```diff
@@ -206,9 +205,9 @@ protected void initEntityTypeField() {
         }
         entityTypeField.setItemLabelGenerator(options::get);
         //noinspection unchecked
-        entityTypeField.setItems(DataProviderUtils.createCallbackDataProvider(options.keySet().stream()
+        entityTypeField.setItems(options.keySet().stream()
```

Comment:
I can explain why did i use dataprovider. When we used previous version of vaadin, comboboxes didn't work with list items normally or ListDataProvider. 

If right now this working well, we can do it. 

I don't think, that we need in any of DataProvider types(limit/offset paginations inside its api). So, if this tested and working fine, merge